charts/adoption-frontend/values.yaml

```
@@ -1,5 +1,6 @@
 nodejs:
   image: hmctspublic.azurecr.io/ado/adoption-frontend:latest
+  ingressHost: adoption-frontend.{{ .Values.global.environment }}.platform.hmcts.net
```

you need to bump the chart version as well, and looks like you should remove frontend from the url based on https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/common/adoption/adoption-frontend.yaml#L28

```suggestion
  ingressHost: adoption.{{ .Values.global.environment }}.platform.hmcts.net
```

---

i have already updated chart version in different PR. I think i am going to leave it there as its going to be merged very soon after this.

---

looks like i have to update that now for pipeline to work.  I will do that now.

---

done